db/activities/data_science_101.md

```diff
@@ -3,10 +3,16 @@ name: Data Science 101
 id: data_science_101
 startTime: June 20, 2020 10:30:00
 endTime: June 20, 2020 11:30:00
-mediaType: "meeting_url"
+mediaType: join_url
```

Comment:
Looks like you changed the types for the property. Please change it back to what it was as it might be in different places.

Reply:
i'm a little confused about the quotations, so I've made the change back but without the quotations. Let me know if the quotations are necessary for some reason :thinking: